### PR TITLE
Mhv-49690/landing page phase 1

### PR DESCRIPTION
**Pull request by KyleCardwell**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- Added a feature flag for phase 1 of Secure Messaging and updated the Landing Page with approved copy for when Phase 1 is active.

### [MHV-49690](https://jira.devops.va.gov/browse/MHV-49690) - Initial Phase 1 Changes ###

> Sketch links (desktop and mobile):
> https://www.sketch.com/s/412905e2-d506-49ce-8d0b-6684526e9fb7/a/uuid/02D99D2B-22FE-4E76-BB02-DDC959E8C4B3
> https://www.sketch.com/s/412905e2-d506-49ce-8d0b-6684526e9fb7/a/uuid/82F18363-65F3-40CD-839F-DC270366A7DF
> 
> Approved text (below and [in slack](https://dsva.slack.com/docs/T03FECE8V/F05TXQY9KQ9)):
> _____________________
> What to know as you try out this tool
> 
> You’re using the new version of My HealtheVet secure messaging on [VA.gov](http://va.gov/). And we need your feedback to help us keep making this tool better for you and all Veterans.
> 
> Send us your feedback and questions using the Feedback button on this page.
> **
> 
> Note: You still have access to the previous version of secure messaging. You can go back to that version at any time. [Go back to the previous version of secure messaging](https://mhv-syst.myhealth.va.gov/mhv-portal-web/secure-messaging)
> 
> _____________________
> 
> Story 1:
> Prep Initial SM Landing Page for Phase 1
> 
> -Enlarge start new message link on Landing Page - dependent on [MHV-49871](https://jira.devops.va.gov/browse/MHV-49871) from UCD
> 
> -Make link/escape hatch back to classic SM more prominent
> -Remove email address
> -Update copy
> -No breadcrumbs
> ***Need assets for all
> Attachment is DRAFT ONLY

## Testing done

- Manual testing to verify changes to the copy when the toggle is turned on/off
- unit tests to validate changes to the copy when the toggle is turned on/off

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |Screenshot 2023-10-11 at 1 57 23 PM|Screenshot 2023-10-11 at 1 51 41 PM|
| Desktop |Screenshot 2023-10-11 at 1 57 35 PM|Screenshot 2023-10-11 at 1 51 57 PM|
| Desktop |Screenshot 2023-10-11 at 1 57 51 PM|Screenshot 2023-10-11 at 3 29 21 PM|
| Desktop |Screenshot 2023-10-11 at 1 58 02 PM|Screenshot 2023-10-11 at 1 52 19 PM|
| Desktop |Screenshot 2023-10-11 at 1 58 10 PM|Screenshot 2023-10-11 at 1 52 28 PM|

## What areas of the site does it impact?

VA.gov - Secure Messaging

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv/secure-messaging/components/FrequentlyAskedQuestions.jsx [hunk 1]
```diff
@@ -92,15 +225,15 @@ const FrequentlyAskedQuestions = ({ prefLink }) => {
...
+          Will my messages become part of my VA medical record?
           </h3>
           <p>
-            This depends. Your care team may decide to include certain messages
-            in your medical record.
+          This depends. Your care team may decide to include certain messages
+          in your medical record.
           </p>
-        </va-accordion-item>
+          </va-accordion-item>
```
Comment (oleksii-morgun-va) [inline src/applications/mhv/secure-messaging/components/FrequentlyAskedQuestions.jsx:236]: if this is not needed, we can probably just delete

File: src/applications/mhv/secure-messaging/components/Dashboard/WelcomeMessage.jsx [hunk 2]
```diff
@@ -1,22 +1,46 @@
...
-      <p>
-        We’re giving the trusted My HealtheVet secure messaging tool a new home
-        on VA.gov. And we need your feedback to help us keep making this tool
-        better for you and all Veterans.
-      </p>
-      <p>
-        Email your feedback and questions to us at <FeedbackEmail />.
-      </p>
+
+      {mhvSecureMessagingToPhase1 ? (
```
Comment (oleksii-morgun-va) [inline src/applications/mhv/secure-messaging/components/Dashboard/WelcomeMessage.jsx:20]: we may need unit tests to cover these conditions

File: src/applications/mhv/secure-messaging/components/FrequentlyAskedQuestions.jsx [hunk 3]
```diff
@@ -1,9 +1,142 @@
...
+import CrisisLineConnectButton from './CrisisLineConnectButton';
 
 const FrequentlyAskedQuestions = ({ prefLink }) => {
-  return (
+  const mhvSecureMessagingToPhase1 = useSelector(
+    state =>
+      state.featureToggles[FEATURE_FLAG_NAMES.mhvSecureMessagingToPhase1],
+  );
+
+  return mhvSecureMessagingToPhase1 ? (
```
Comment (oleksii-morgun-va) [inline src/applications/mhv/secure-messaging/components/FrequentlyAskedQuestions.jsx:14]: we may need unit tests to cover these conditions

**Review by vsaleem — APPROVED** [review]
Perhaps at some point we could put all of this text into constants.  Along with Oleksii's recs this looks good.

**Review by alex-morgun-1 — APPROVED** [review]

**Review by alex-morgun-1 — APPROVED** [review]

**Review by ataker — APPROVED** [review]

**Pull request merged**